backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/validator/UserProfileValidator.java

```diff
@@ -40,14 +40,19 @@ public ValidationResult propertyProvided(UserProfileProperty property) {
                 .when(property == null);
     }
 
-    public ValidationResult authorized(DbUser currentUser, Guid targetUserId) {
+    public ValidationResult authorizedToModify(DbUser currentUser, Guid targetUserId, boolean isSuperUser) {
```

Comment:
`isSuperUser` is not really correct as you call this method below from the method `authorized` where the user is only `admin`. What about something like `sufficientAdminRights`?

Also, I would keep the method's name to `authorized` as it is quite general and is not used just for modification.